lib/paypal_express/api.rb

```diff
@@ -352,6 +352,10 @@ def initiate_express_checkout(kb_account_id, amount, currency, properties, conte
         options = {}
         options[:return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :return_url)
         options[:cancel_return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :cancel_return_url)
+        if properties_hash[:max_amount]
```

Comment:
For consistency, we should use [Utils.normalized](https://github.com/killbill/killbill-plugin-framework-ruby/blob/a0cfeb8d121a189249ea896e4763ba764a07b58c/lib/killbill/helpers/active_merchant/utils.rb#L34-L43) as well.